### PR TITLE
Public repo

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
New PR in favor of #1190

http://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/customization.html
http://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/public.html

cc: @hflynn

### CODE REVIEW

File: omero/sysadmins/customization.txt [hunk 1]
```diff
@@ -0,0 +1,119 @@
+Customize your public repository
+================================
+
+The OMERO clients offer flexible user interface that can be customized.
+The sections below describe how to set up these features.
+
+Index Page
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: Page -> page

File: omero/sysadmins/customization.txt [hunk 2]
```diff
@@ -0,0 +1,119 @@
+Customize your public repository
+================================
+
+The OMERO clients offer flexible user interface that can be customized.
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: offer **a** flexible - need to add a word here

File: omero/sysadmins/customization.txt [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+The sections below describe how to set up these features.
+
+Index Page
+----------
+
+Create new custom template in /your/path/to/templates/mytemplate/index.html
+and add the following
+
+.. note::
+    User will no longer be automaticaly redirected to the login page
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: User -> Users 
automatically

File: omero/sysadmins/customization.txt [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+::
+
+    $ bin/omero config set omero.web.template_dirs '/your/path/to/templates/'
+    $ bin/omero config set omero.web.index_template 'mytemplate/index.html'
+
+.. figure:: /images/indexPage.png
+
+
+Login Page
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: Page -> page

File: omero/sysadmins/customization.txt [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+----------
+
+:property:`omero.web.login_logo` allows to customize webclient login page
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: allows **you** to customize **the** webclient

File: omero/sysadmins/customization.txt [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+:property:`omero.web.login_redirect` property redirects to the givin location
+after loging in.
+
+::
+
+    $ bin/omero config set omero.web.login_redirect '{"redirect": ["webindex"], "viewname": "load_template", "args":["userdata"], "query_string": "experimenter=-1"}'
+
+
+
+Top Links Menu
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: Links Menu -> links menu

File: omero/sysadmins/customization.txt [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+shouldn’t display.
+
+::
+
+    $ bin/omero config append omero.client.scripts_to_ignore "/my_scripts/script.py"
+
+.. figure:: /images/disableScripts.png
+
+
+Download Restricitons
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: Restricitons -> restrictions

File: omero/sysadmins/customization.txt [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+    $ bin/omero config set omero.web.login_logo 'http://www.url/to/image.png'
+
+.. figure:: /images/customLogin.png
+
+
+Login redirection
+-----------------
+
+:property:`omero.web.login_redirect` property redirects to the givin location
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: givin -> given

File: omero/sysadmins/customization.txt [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+after loging in.
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: loging -> logging

File: omero/sysadmins/customization.txt [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "webfigure"]'
+    $ bin/omero config set omero.web.ui.top_links '["GRE", "http://lifesci.dundee.ac.uk/gre"]'
+
+.. figure:: /images/topLink.png
+
+
+Group and Users in dropdown menu
+--------------------------------
+
+Customize group and users drop down menu by changing labels or hiding entire
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: Customize **the** group**s** and users dropdown menu by changing **the** labels or hiding **the** entire list.

File: omero/sysadmins/customization.txt [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+    $ bin/omero config set omero.client.ui.menu.dropdown.all "All Members"
+    $ bin/omero config set omero.client.ui.menu.dropdown.colleagues.enabled true
+
+.. figure:: /images/dropdownMenu.png
+
+
+Orphaned container
+------------------
+
+:property:`omero.client.ui.tree.orphans.name` allows to changes name of
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: allows **you** to change **the** name of **the** "Orphaned images" container located in **the** client data manager tree.

File: omero/sysadmins/customization.txt [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+
+    $ bin/omero config set omero.client.ui.tree.orphans.name "Orphaned images"
+
+.. figure:: /images/orphans.png
+
+
+Disabling scripts
+-----------------
+
+:property:`omero.client.scripts_to_ignore` hides list of scripts Clients
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: hides **the** scripts **that the** clients should not display

File: omero/sysadmins/customization.txt [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+
+    $ bin/omero config append omero.client.scripts_to_ignore "/my_scripts/script.py"
+
+.. figure:: /images/disableScripts.png
+
+
+Download Restricitons
+---------------------
+
+:property:`omero.policy.binary_access` determine whether users can access
```
Comment (hflynn) [inline omero/sysadmins/customization.txt]: determine**s**

**Comment by hflynn** [comment]
Ok, the customization page needs some tidying (N.B. use of bold is to help make my corrections clearer, not to suggest you make some of the text bold) and I think given we have 2 new pages now, we could probably split this into it's own section on the index page to make it easier to find:

What about
"**Using OMERO as a Public Data Repository**

This chapter explains how to configure your OMERO server to host data for external users to access."

Underneath the current configuration section and above the data import and storage section.

File: omero/sysadmins/index.txt [hunk 14]
```diff
@@ -59,6 +59,19 @@ performance and security.
...
+Optimizing and Publishing Data Repository
+*****************************************
+
+This chapter explains how to customize your OMERO data repository.
+
+.. toctree::
+    :maxdepth: 2
+    :titlesonly:
+
+    customization
```
Comment (hflynn) [inline omero/sysadmins/index.txt]: You need public first here

File: omero/sysadmins/index.txt [hunk 15]
```diff
@@ -59,6 +59,19 @@ performance and security.
     omero-home-prefix
     config
 
+*****************************************
+Optimizing and Publishing Data Repository
+*****************************************
+
+This chapter explains how to customize your OMERO data repository.
```
Comment (hflynn) [inline omero/sysadmins/index.txt]: …customize the appearance and functionality of OMERO clients to host images for groups or public viewing.

File: omero/sysadmins/index.txt (hunk 14 above)
Comment (aleksandra-tarkowska) [inline omero/sysadmins/index.txt]: as agreed public is a part of customization, comment sorted below

**Comment by hflynn** [comment]
Sorry @aleksandra-tarkowska but I'm still not happy with the section title - what about:
'**Optimizing OMERO as a Data Repository**'

**Comment by hflynn** [comment]
Great! Merging before we get more bogged down with this

**Pull request merged**